supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -11,5 +11,29 @@
 [[admonitions]]
 == Admonitions
 
+Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions in close proximity to one another. If multiple admonitions are necessary, restructure the information by removing the less important statements and placing them into the flow of the main content.
+
+Admonitions include:
```

Comment:
I think that AsciiDoc supports more admonition types than this, so we should be more specific about that this is the list that we want to use. If I remember correctly, I don't think that the "CAUTION" type is supported by the portal.

Maybe "Allowed admonition types:"

Reply:
"CAUTION" is not fully supported by our tooling and does not render fully styled. Please see CCS-1861 in the dxp content tooling jira for the discussion. As of this moment, I would recommend we only provide guidance on fully supported admonitions and note that CAUTION is not supported.